java-compute/.OwlBot.yaml

```diff
@@ -27,7 +27,7 @@ deep-copy-regex:
   dest: "/owl-bot-staging/java-compute/$1/proto-google-cloud-compute-$1/src"
 - source: "/google/cloud/compute/(v\\d)/.*-java/gapic-google-.*/src"
   dest: "/owl-bot-staging/java-compute/$1/google-cloud-compute/src"
-- source: "/google/cloud/compute/(v.*)/.*-java/samples/snippets/generated"
```

Comment:
Would you explain why the previous regular expression didn't match the expected files?

Reply:
Sorry, just double checked and you're right - the previous expression did work. It looks like the update PR just hasn't been merged in yet (https://github.com/googleapis/google-cloud-java/pull/9262/files). I will close this.